### ISSUE TITLE
list of keys?

### ISSUE THREAD

**Issue by ReK2Fernandez** [way-cooler/way-cooler#463]
I cant find a way to logout back to sddm...
Is there a list somewhere? also the way-cooler-bg will just open awindow with the background instead of changing the background, also I think is doing the amazing setup not the i3 how can I switch this? I install with yaourt/pacaur from the arch AUR the git version

**Comment by ReK2Fernandez** [comment]
Ok I found some keys on the main config in /etc/ so that part is better, but still does not say how to log out back to login window.. only restart way-cooler.

**Comment by ReK2Fernandez** [comment]
ok so there hidden in gray letters there is how to log out :) so I just need a way to change the background and I am all set :)

**Comment by Timidger** [comment]
Hey @ReK2Fernandez.

You are probably using the latest version of the background program, which requires the latest version of Way to (eg, the master version, _not_ 0.6.2). Try using either an older version of the background program or compile Way Cooler from source at the latest version on github.

**Comment by ReK2Fernandez** [comment]
@Timidger Thanks for the response... I am using the git version in the arch AUR, I though the -git will always get the latest from github? cheers

**Comment by Timidger** [comment]
You might be using an old version of the background program then, please compile it from source since I think the AUR version of the background program is out of date

**Comment by ReK2Fernandez** [comment]
@Timidger ok I compiled it and got it working... atr first I created a pacage with the latest tag and still was having the issue so I just cloned the master and compiled that added it to my path and works now.  thanks.. I will try to create a package that grabs the master, but wont have time until after CCC in Germany. ohh BTW how do I lets say run apps that need to run all the time but dont necesary need to be visible on windows just an icon? like keybase run_keybase if not it wont mount the filesystem etc.
same with password managers like enpass for example.. what will be the best way?

thanks
